awx/ui_next/src/components/AddDropDownButton/AddDropDownButton.jsx

```diff
@@ -1,11 +1,10 @@
 import React, { useState, useRef, useEffect } from 'react';
 import { Link, withRouter } from 'react-router-dom';
 import { withI18n } from '@lingui/react';
-import { t } from '@lingui/macro';
 import { Dropdown, DropdownPosition } from '@patternfly/react-core';
 import { ToolbarAddButton } from '@components/PaginatedDataList';
 
-function TemplateAddButton({ match, i18n }) {
+function AddDropDownButton({ topUrl, bottomUrl, topLabel, bottomLabel }) {
```

Comment:
Instead of hard-coding a "top" and "bottom" link, this should take an array of link values, so any number of items can be put in the dropdown. Structure it something like:

```
[{
  url: 'inventory/add',
  label: 'Add Inventory',
}, {
  url: 'smart_inventory/add',
  label: 'Add Smart Inventory',
}]
```